dcpy/test/connectors/edm/test_publishing.py

```diff
@@ -50,6 +50,7 @@ def test_upload(create_buckets, create_temp_filesystem, mock_data_constants):
 
 
 def test_publish(create_buckets, create_temp_filesystem, mock_data_constants):
+    """Tests publish function as well as get_version and get_latest_version"""
```

Comment:
Expand the rest of this function to see how get_version and get_latest_version is tested here. Not the most unit of tests given that it's also testing publish, but it's still fairly complete in terms of what we'd want tested